src/index.js

```diff
@@ -1,14 +1,103 @@
-import _ from 'lodash';
 import './style.css';
+import TasksManager from './tasks_manager';
+import DisplayManager from './display_manager';
 
-function component() {
-  const element = document.createElement('div');
+const selector = (element) => document.querySelector(element);
+const selectorAll = (element) => document.querySelectorAll(element);
+const input = selector('input');
+const manager = new TasksManager();
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'ahmed'], ' ');
-  element.classList.add('hello');
+const updateTask = (event, index, focus = false) => {
+  if (event.key === 'Enter' || focus) {
+    manager.updateTask(
+      index,
+      selector(`.task_${index}`).value,
+      selector(`.task_${index}_checkbox`).checked,
+    );
+    selector('.add_task input').focus();
+  }
+};
 
-  return element;
-}
+const updateStatus = (index) => {
+  manager.updateStatus(index, selector(`.task_${index}_checkbox`).checked);
+  selector(`.task[data-id='${index}'] .input_task`).classList.toggle(
+    'completed',
+  );
+};
 
-document.body.appendChild(component());
+const display = () => {
+  DisplayManager.reset(selector('.tasks'));
+
+  manager
+    .getTasks()
+    .forEach((task) => DisplayManager.displayTask(selector('.tasks'), task));
+
+  selectorAll('.fa-trash').forEach((e) => e.addEventListener('click', () => {
+    // eslint-disable-next-line no-use-before-define
+    deleteTask(e.dataset.id);
+  }));
+
+  selectorAll('.input_task').forEach((e) => e.addEventListener('keyup', (event) => {
+    console.log(e.dataset);
+    updateTask(event, e.dataset.id);
+  }));
+
+  selectorAll('.checkbox_task').forEach((e) => e.addEventListener('change', () => {
+    console.log(e.dataset.id);
+    updateStatus(e.dataset.id);
+  }));
```

Comment:
- [ ] [OPTIONAL] please remove the 2 `console.log` it will make your application work better and your codebase will look a lot cleaner.

Reply:
Hi @mikethreels 
I forgot those thanks i will remove them now